Dockerfile.devkit

```diff
@@ -107,6 +107,10 @@ COPY --from=docker /usr/local/bin/docker /usr/local/bin/
 COPY --from=govc /govc /usr/local/bin/
 COPY --from=builder /tools /usr/local/bin
 
+ARG CONTAINERD_VERSION
+ADD "https://github.com/containerd/containerd/releases/download/v${CONTAINERD_VERSION}/cri-containerd-cni-${CONTAINERD_VERSION}-linux-amd64.tar.gz" /opt/dkp/containerd.tar.gz
+# FIXME(dkoshkin) use a real FIPS version
+ADD "https://github.com/containerd/containerd/releases/download/v${CONTAINERD_VERSION}/cri-containerd-cni-${CONTAINERD_VERSION}-linux-amd64.tar.gz" /opt/dkp/containerd-fips.tar.gz
```

Comment:
Is there a way we can do this all in ansible where its either a URL or if `offline` in the `/artifacts` dir?

Reply:
We can and thats how I started in https://github.com/mesosphere/konvoy-image-builder/pull/329/commits/3acc99e30198bec90c6f5163307bd3264db30736

But I've been thinking about this, currently we install the Containerd package from the same offline repo that has Kubernetes packages. This basically couples KIB to a specific version of Kubernetes, Containerd, `dkp` since all these things need to align to work.
Moving this responsibility to KIB greatly simplifies all of it as now users would only need to worry about the Kubernetes version and a version of KIB should be able to work with any combination of Kubernetes and `dkp`.

Reply:
Hrm, I see, the hozer though is that since the version is still an ansible variable, it can be overridden anyway, and then it will fail because the file isn't baked into the docker layer, and this inflates the size of the docker image by ~50MB.

I totally agree that KIB should be responsible for the pairing, I just think that the ansible portion should be the one doing that, since at the end of the day, packer is just invoking ansible to actually do the provisioning. I dont' think we should be coupling to the use of a particular docker image. In theory there's no reason not to be able to use any devkit version since all its for is making sure packer, ansible, and goss are installed in an "easy" way.

Not a blocker just my $0.02 ;)

Reply:
Talking about this in standup, this approach may not be good as we are going to have multiple different tars for each OS/version/FIPS combination.